tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -166,16 +169,30 @@ def efetch(self, db, query_key, webenv):
             try:
                 response = urllib2.urlopen(req)
                 fasta = response.read()
```

Comment:
I think a better solution is to use
```
if response.getcode() == 200:
    fasta = response.read()
    serverTransaction = True
else:
    continue
```

Normally that should cover all the exceptions you're trying to catch later on,
but I haven't tested this.